### PR TITLE
Translate catalog volume monitoring screen

### PR DESCRIPTION
**Pull request by ahocquard**

### CODE REVIEW

File: content/md/administration/catalog-volume-monitoring_fr.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+id: catalog-volume-monitoring
+themes: administration, akeneo-concepts, catalog-settings 
+title: Controllr la **taille de votre catalogue** (since 2.3)
```
Comment (ahocquard) [inline content/md/administration/catalog-volume-monitoring_fr.md]: Controller

File: content/md/administration/catalog-volume-monitoring_fr.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+Cet écran contient par exemple le nombre de familles ou le nombre moyen d'attributs par famille.
+
+![Catalog volume monitoring dashboard](../img/Activity_CatalogVolumeMonitoring_fr.png)
+
+::: warning
+Cet écran n'est disponible que depuis la version 2.3 du PIM.
+:::
+
+# Où est-ce?
```
Comment (ahocquard) [inline content/md/administration/catalog-volume-monitoring_fr.md]: Any idea about a better formulation ?

File: content/md/administration/catalog-volume-monitoring_fr.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+# Comment comprendre ce dashboard ?
+
+## Les différentes sections
+
+L'écran `Volume du catalogue` est composé de plusieurs sections.
+
+### Le nombre de valeurs de produit
+
+La première valeur affichée dans le widget représente le nombre total de [valeurs de produit](/articles/what-is-a-product-value.html) contenues dans le PIM.
```
Comment (ahocquard) [inline content/md/administration/catalog-volume-monitoring_fr.md]: links should be translated or not?

File: content/md/administration/catalog-volume-monitoring_fr.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+C'est un indicateur de la valeur des données présentes dans votre catalogue produit. 
+
+![Widget du nombre de valeurs de produit](../img/Activity_CatalogVolumeMonitoring_PvWidget_fr.png)
+
+Nous vous encourageons à surveiller cette valeur régulièremeent. Plus le nombre de valeurs de produit est important, plus votre PIM vous est utile !
+
+Nous avons écris un [article](https://medium.com/akeneo-labs/because-your-product-catalog-typology-matters-e1a9af4c33e0) en anglais afin d'expliquer pourquoi le nombre de valeurs produit est une métrique intéressante pour évaluer la taille de votre catalogue. Ne soyez pas effrayé par la taille de cet article ! Il mettra en lumière toute vos interrogations. Vous ne parlerez plus de la taille de votre catalogue de la même façon ;)
```
Comment (ahocquard) [inline content/md/administration/catalog-volume-monitoring_fr.md]: valeurs de produit

Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: `Il mettra en lumière toute vos interrogations` is a little bit too much.
What about `, il répondra à toute vos questions`

File: content/md/administration/catalog-volume-monitoring_fr.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+::: tips
+Il se peut que le widget ne soit pas visible.
+Si votre PIM est auto-hébergé ou bien en PAAS, c'est probablement parce que vous ou votre administrateur n'avez pas lancé le job quotidien de afin de calculer les valeurs du widget.
```
Comment (ahocquard) [inline content/md/administration/catalog-volume-monitoring_fr.md]: de afin de

File: content/md/administration/catalog-volume-monitoring_fr.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+Dans l'édition entreprise, une section additonnelle `Axes des assets` est présente. Celle-ci renseigne sur sur la volumétrie des [assets](/articles/work-with-assets.html) dans le PIM.
+In the Entreprise Edition, there is an additional section called `Axes on assets` that will give you the volume regarding the [assets](/articles/work-with-assets.html).
+:::
+
+## Les alertes
+
+Sur certains axes, vous observerez éventuellement des alertes. Celles-ci s'affichent derrière la métrique et l'icône de l'axe est alors affiché en violet. 
+![Alerte dans l'écran de volumétrie du catalogue](../img/Activity_CatalogVolumeMonitoring_Alert_fr.png)
+
+Cette alerte que la volumétrie correspondant à l'axe est élevée. Si vous rencontrez un problème concernant cet axe, veuillez nous contacter.
```
Comment (ahocquard) [inline content/md/administration/catalog-volume-monitoring_fr.md]: indique

File: content/md/administration/catalog-volume-monitoring_en.md [hunk 7]
```diff
@@ -22,16 +22,16 @@ This screen is only available starting the 2.3 version of the PIM.
 
 To access this amazing (;)) screen, follow these very simple steps:
 1. Click on the `Activity` menu
-1. Click on the `Catalog volume monitoring` menu entry
-1. Enjoy!
+2. Click on the `Catalog volume monitoring` menu entry
```
Comment (NolwennP) [inline content/md/administration/catalog-volume-monitoring_en.md]: You don't need to change that. Markdown automatically adapts the numbers correctly. :)

File: content/md/administration/catalog-volume-monitoring_en.md [hunk 8]
```diff
@@ -43,7 +43,7 @@ We wrote an [article](https://medium.com/akeneo-labs/because-your-product-catalo
 
 ::: tips
 You can't see the widget we are talking about? What a shame!  
-If you are a Community Edition user or working on the Premise/PAAS version, it's probably because you or your administrator didn't launch the CRON job to calculate the values shown in this widget.  
+If your PIM is on Premise or on our PAAS version, it's probably because you or your administrator didn't launch the CRON job to calculate the values shown in this widget.
```
Comment (NolwennP) [inline content/md/administration/catalog-volume-monitoring_en.md]: Why did you remove the Community Edition version? It's also true with the CE version.

File: content/md/administration/catalog-volume-monitoring_fr.md (hunk 1 above)
Comment (NolwennP) [reply content/md/administration/catalog-volume-monitoring_fr.md]: Suivre la *taille de votre catalogue** (depuis la v2.3)

File: content/md/administration/catalog-volume-monitoring_fr.md [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+id: catalog-volume-monitoring
+themes: administration, akeneo-concepts, catalog-settings 
+title: Controllr la **taille de votre catalogue** (since 2.3)
+popular: false
+related: what-is-a-product-value, what-is-an-attribute, what-is-a-family, what-is-a-channel, what-is-a-locale, what-is-a-category
+---
+
+# Un écran dédié
+
+Un écran est disponible dans le PIM afin de controller la taille du catalogue sur plusieurs axes.
```
Comment (NolwennP) [inline content/md/administration/catalog-volume-monitoring_fr.md]: afin de suivre

File: content/md/administration/catalog-volume-monitoring_fr.md [hunk 10]
```diff
@@ -0,0 +1,72 @@
...
+
+Cet écran contient par exemple le nombre de familles ou le nombre moyen d'attributs par famille.
```
Comment (NolwennP) [inline content/md/administration/catalog-volume-monitoring_fr.md]: Cet écran vous donne par exemple

File: content/md/administration/catalog-volume-monitoring_fr.md (hunk 2 above)
Comment (NolwennP) [reply content/md/administration/catalog-volume-monitoring_fr.md]: Où trouver cet écran ?

File: content/md/administration/catalog-volume-monitoring_fr.md [hunk 11]
```diff
@@ -0,0 +1,72 @@
...
+::: warning
+Cet écran n'est disponible que depuis la version 2.3 du PIM.
+:::
+
+# Où est-ce?
+
+Voici les étapes afin d'accéder à ce superbe (;)) écran:
+
+1. Cliquer sur le menu `Activité`
+2. Cliquer sur le sous-menu `Volume du catalogue`
```
Comment (NolwennP) [inline content/md/administration/catalog-volume-monitoring_fr.md]: `Volumétrie du catalogue`

File: content/md/administration/catalog-volume-monitoring_fr.md [hunk 12]
```diff
@@ -0,0 +1,72 @@
...
+3. Et voilà !
+
+# Comment comprendre ce dashboard ?
+
+## Les différentes sections
+
+L'écran `Volume du catalogue` est composé de plusieurs sections.
```
Comment (NolwennP) [inline content/md/administration/catalog-volume-monitoring_fr.md]: `Volumétrie du catalogue`

File: content/md/administration/catalog-volume-monitoring_fr.md [hunk 13]
```diff
@@ -0,0 +1,72 @@
...
+
+# Où est-ce?
+
+Voici les étapes afin d'accéder à ce superbe (;)) écran:
+
+1. Cliquer sur le menu `Activité`
+2. Cliquer sur le sous-menu `Volume du catalogue`
+3. Et voilà !
+
+# Comment comprendre ce dashboard ?
```
Comment (NolwennP) [inline content/md/administration/catalog-volume-monitoring_fr.md]: Comment lire ce tableau de bord ?

File: content/md/administration/catalog-volume-monitoring_fr.md (hunk 3 above)
Comment (NolwennP) [inline content/md/administration/catalog-volume-monitoring_fr.md]: Remove "dans le widget" cause there is not really a widget.

Comment (NolwennP) [reply content/md/administration/catalog-volume-monitoring_fr.md]: `/articles/fr/what-is-a-product-value.html`
If this article is not translated yet, it will redirect the user to a 404 page asking him to help with the translation.

Comment (NolwennP) [inline content/md/administration/catalog-volume-monitoring_fr.md]: Why did you remove the fact that this is the most important info?

File: content/md/administration/catalog-volume-monitoring_fr.md [hunk 14]
```diff
@@ -0,0 +1,72 @@
...
+
+## Les différentes sections
+
+L'écran `Volume du catalogue` est composé de plusieurs sections.
+
+### Le nombre de valeurs de produit
+
+La première valeur affichée dans le widget représente le nombre total de [valeurs de produit](/articles/what-is-a-product-value.html) contenues dans le PIM.
+C'est un indicateur de la valeur des données présentes dans votre catalogue produit.
```
Comment (NolwennP) [inline content/md/administration/catalog-volume-monitoring_fr.md]: C'est un très bon indicateur de la valeur finale de votre PIM.

File: content/md/administration/catalog-volume-monitoring_fr.md (hunk 4 above)
Comment (NolwennP) [inline content/md/administration/catalog-volume-monitoring_fr.md]: écrit

Comment (NolwennP) [inline content/md/administration/catalog-volume-monitoring_fr.md]: (en anglais) in parenthesis

Comment (NolwennP) [inline content/md/administration/catalog-volume-monitoring_fr.md]: toutes vos interrogations

Comment (NolwennP) [inline content/md/administration/catalog-volume-monitoring_fr.md]: la même façon.

File: content/md/administration/catalog-volume-monitoring_fr.md [hunk 15]
```diff
@@ -0,0 +1,72 @@
...
+C'est un indicateur de la valeur des données présentes dans votre catalogue produit. 
+
+![Widget du nombre de valeurs de produit](../img/Activity_CatalogVolumeMonitoring_PvWidget_fr.png)
+
+Nous vous encourageons à surveiller cette valeur régulièremeent. Plus le nombre de valeurs de produit est important, plus votre PIM vous est utile !
+
+Nous avons écris un [article](https://medium.com/akeneo-labs/because-your-product-catalog-typology-matters-e1a9af4c33e0) en anglais afin d'expliquer pourquoi le nombre de valeurs produit est une métrique intéressante pour évaluer la taille de votre catalogue. Ne soyez pas effrayé par la taille de cet article ! Il mettra en lumière toute vos interrogations. Vous ne parlerez plus de la taille de votre catalogue de la même façon ;)
+
+::: tips
+Il se peut que le widget ne soit pas visible.
```
Comment (NolwennP) [inline content/md/administration/catalog-volume-monitoring_fr.md]: Add two space at the end, so that Markdown really go to the next line.

File: content/md/administration/catalog-volume-monitoring_fr.md (hunk 5 above)
Comment (NolwennP) [reply content/md/administration/catalog-volume-monitoring_fr.md]: qui permet de

File: content/md/administration/catalog-volume-monitoring_fr.md [hunk 16]
```diff
@@ -0,0 +1,72 @@
...
+Si votre PIM est auto-hébergé ou bien en PAAS, c'est probablement parce que vous ou votre administrateur n'avez pas lancé le job quotidien de afin de calculer les valeurs du widget.
+
+Afin de lancer ce job, veuillez vous référer à notre documentation technique [de la CE] (https://docs.akeneo.com/2.3/install_pim/manual/installation_ce_archive.html#configuring-tasks-via-crontab) ou bien [de la EE](https://docs.akeneo.com/2.3/install_pim/manual/installation_ee_archive.html#configuring-tasks-via-crontab).
+:::
+
+### Les autres sections
+
+La prochaine section `Axes qui influencent le nombre de valeurs de produit` est dédié à plusieurs axes qui ont un impact sur le nombre de valeurs de produit total.
```
Comment (NolwennP) [inline content/md/administration/catalog-volume-monitoring_fr.md]: La section suivante

Comment (NolwennP) [reply content/md/administration/catalog-volume-monitoring_fr.md]: est dédié à l'ensemble des axes

File: content/md/administration/catalog-volume-monitoring_fr.md [hunk 17]
```diff
@@ -0,0 +1,72 @@
...
+Par exemple, plus vous avez de canaux, plus le nombre de valeurs de produit sera élevé.
+
+La section `Axes liés à la structure de votre catalogue` rassemble tous les axes ayant un impact sur la structure du catalogue, tel que le nombre de familles.
+
+Enfin, la section `Axes concernant la modélisation des produits avec variantes` affiche le nombre de product models et le nombre de [produit variants](/articles/what-about-products-variants.html).
```
Comment (NolwennP) [inline content/md/administration/catalog-volume-monitoring_fr.md]: le nombre de modèles de produit et le...

Comment (NolwennP) [inline content/md/administration/catalog-volume-monitoring_fr.md]: Don't forget to add /fr/ into the link

File: content/md/administration/catalog-volume-monitoring_fr.md [hunk 18]
```diff
@@ -0,0 +1,72 @@
...
+
+::: ee
+Dans l'édition entreprise, une section additonnelle `Axes des assets` est présente. Celle-ci renseigne sur sur la volumétrie des [assets](/articles/work-with-assets.html) dans le PIM.
```
Comment (NolwennP) [inline content/md/administration/catalog-volume-monitoring_fr.md]: Don't forget to add /fr/ into the link

Comment (NolwennP) [inline content/md/administration/catalog-volume-monitoring_fr.md]: édition Entreprise

Comment (NolwennP) [inline content/md/administration/catalog-volume-monitoring_fr.md]: additionnelle `Axes sur les ressources`

File: content/md/administration/catalog-volume-monitoring_fr.md [hunk 19]
```diff
@@ -0,0 +1,72 @@
...
+In the Entreprise Edition, there is an additional section called `Axes on assets` that will give you the volume regarding the [assets](/articles/work-with-assets.html).
```
Comment (NolwennP) [inline content/md/administration/catalog-volume-monitoring_fr.md]: To remove

File: content/md/administration/catalog-volume-monitoring_fr.md (hunk 6 above)
Comment (NolwennP) [inline content/md/administration/catalog-volume-monitoring_fr.md]: `_Qu'est-ce que signifie cette alerte ?_`

Comment (NolwennP) [inline content/md/administration/catalog-volume-monitoring_fr.md]: Si vous rencontrez un problème à cause de cette volumétrie, n'hésitez pas à nous contacter.

Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: you have already a chapter for it, so it's quite useless no? (even for english version btw)

Comment (NolwennP) [reply content/md/administration/catalog-volume-monitoring_fr.md]: It's just for the article to be less boring.

File: content/md/administration/catalog-volume-monitoring_fr.md (hunk 3 above)
Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: I don't like "L'information la plus importante" as a title. It's kind of weird. It seems as a "bait title" and it indicates nothing.
I prefer to add a sentence.


Also, is it really the most important information? Maybe it's the completeness of the catalog after all.

Comment (NolwennP) [reply content/md/administration/catalog-volume-monitoring_fr.md]: This is the most important information of the screen yes. Completeness is also important but this is for another purpose.
It's ok if you don't put it as a title. But I think we should really emphasize somewhere on the fact this is the most representative number of your catalog size, so this is the axis we should always have a close look at.

Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: I will add it in the title

File: content/md/administration/catalog-volume-monitoring_en.md (hunk 8 above)
Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_en.md]: It's also true with the enterprise edition.
Actually we are mixing two things: CE/EE and on premise/PAAS. It is two distincts things and it's quite confusing the way it's done.

Comment (NolwennP) [reply content/md/administration/catalog-volume-monitoring_en.md]: It's not true for the EE in Serenity mode. That's why I can't say EE on its own. But maybe we can reformulate.

Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_en.md]: That's why I specify on Premise and PAAS version, neither  SAAS nor EE. This way, you cover all the cases.

Comment (NolwennP) [reply content/md/administration/catalog-volume-monitoring_en.md]: Ok. I thought "on Premise" was EE only. Remove the uppercase "P" in this case.

Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_en.md]: "on premise" means: it's installed and monitored on your internal servers.
Meaning that all CE are "on premise".

Comment (NolwennP) [reply content/md/administration/catalog-volume-monitoring_en.md]: Yep, that's what Gaël made me realized. But still, remove the uppercase "P" because it can make us think that we only talk of the on-premise EE which is called "On Premise". Even better, use "on-premise", it's how it's really called in English.

File: content/md/administration/catalog-volume-monitoring_fr.md (hunk 1 above)
Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: done

File: content/md/administration/catalog-volume-monitoring_fr.md (hunk 2 above)
Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: done

File: content/md/administration/catalog-volume-monitoring_fr.md (hunk 3 above)
Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: done

File: content/md/administration/catalog-volume-monitoring_fr.md (hunk 5 above)
Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: done

File: content/md/administration/catalog-volume-monitoring_fr.md (hunk 6 above)
Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: done

File: content/md/administration/catalog-volume-monitoring_fr.md (hunk 9 above)
Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: done

File: content/md/administration/catalog-volume-monitoring_fr.md (hunk 10 above)
Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: done

File: content/md/administration/catalog-volume-monitoring_fr.md (hunk 11 above)
Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: done

File: content/md/administration/catalog-volume-monitoring_fr.md (hunk 13 above)
Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: that's the question

File: content/md/administration/catalog-volume-monitoring_fr.md (hunk 3 above)
Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: done

Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: done

File: content/md/administration/catalog-volume-monitoring_fr.md (hunk 14 above)
Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: done

File: content/md/administration/catalog-volume-monitoring_fr.md (hunk 4 above)
Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: done

Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: done

Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: done

Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: you are funnier in english :trollface:

File: content/md/administration/catalog-volume-monitoring_fr.md (hunk 17 above)
Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: done

File: content/md/administration/catalog-volume-monitoring_fr.md (hunk 16 above)
Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: done

File: content/md/administration/catalog-volume-monitoring_fr.md (hunk 15 above)
Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: done

File: content/md/administration/catalog-volume-monitoring_fr.md (hunk 17 above)
Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: done

File: content/md/administration/catalog-volume-monitoring_fr.md (hunk 18 above)
Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: done

Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: done

File: content/md/administration/catalog-volume-monitoring_fr.md (hunk 19 above)
Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: done

File: content/md/administration/catalog-volume-monitoring_fr.md (hunk 4 above)
Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: done

File: content/md/administration/catalog-volume-monitoring_fr.md [hunk 20]
```diff
@@ -0,0 +1,73 @@
...
+::: tips
+Il se peut que le widget ne soit pas visible.  
+Si votre PIM est auto-hébergé ou bien en PAAS, c'est probablement parce que vous ou votre administrateur n'avez pas lancé le job quotidien qui permet de calculer les valeurs du widget.
+
+Afin de lancer ce job, veuillez vous référer à notre documentation technique [de la CE] (https://docs.akeneo.com/2.3/install_pim/manual/installation_ce_archive.html#configuring-tasks-via-crontab) ou bien [de la EE](https://docs.akeneo.com/2.3/install_pim/manual/installation_ee_archive.html#configuring-tasks-via-crontab).
+:::
+
+### Les autres sections
+
+La section suivante `Axes qui influencent le nombre de valeurs de produit` est dédié à l'ensemble des axes qui ont un impact sur le nombre de valeurs de produit total.
```
[the remark this answers is not in the document]
Comment (ahocquard) [reply content/md/administration/catalog-volume-monitoring_fr.md]: no, it's not number of total value but total number of values.

File: content/md/administration/catalog-volume-monitoring_fr.md [hunk 21]
```diff
@@ -0,0 +1,73 @@
...
+
+Enfin, la section `Axes concernant la modélisation des produits avec variantes` affiche le nombre de modèles de produit et le nombre de [produit variants](/articles/fr/what-about-products-variants.html).
+
+::: Edition Entreprise
+Dans l'édition entreprise, une section additonnelle `Axes sur les ressources` est présente. Celle-ci renseigne sur sur la volumétrie des [assets](/articles/fr/work-with-assets.html) dans le PIM.
+:::
+
+## Les alertes
+
+Sur certains axes, vous observerez éventuellement des alertes. Celles-ci s'affichent derrière la métrique et l'icône de l'axe est alors affiché en violet.
```
[the remark this answers is not in the document]
Comment (LaureBro) [reply content/md/administration/catalog-volume-monitoring_fr.md]: _En informatique, une icône est une petite image symbolisant un élément, comme un programme ou un document, auquel elle est associée._ c'est féminin :)

File: content/md/administration/catalog-volume-monitoring_fr.md [hunk 22]
```diff
@@ -0,0 +1,73 @@
...
+L'écran `Volumétrie du catalogue` est composé de plusieurs sections.
+
+### Le nombre de valeurs de produit : l'information la plus importante
+
+La première valeur affichée représente le nombre total de [valeurs de produit](/articles/fr/what-is-a-product-value.html) contenues dans le PIM.
+C'est un très bon indicateur de la véritable valeur de votre PIM.
+
+![Widget du nombre de valeurs de produit](../img/Activity_CatalogVolumeMonitoring_PvWidget_fr.png)
+
+Nous vous encourageons à surveiller cette valeur régulièremeent. Plus le nombre de valeurs de produit est important, plus votre PIM vous est utile !
```
[the remark this answers is not in the document]
Comment (NolwennP) [reply content/md/administration/catalog-volume-monitoring_fr.md]: Nope, it means that if you enriched a lot of data, you will have more product values, so that means that you really need a PIM (otherwise you'll still be enriching in multiple spreadsheets and have all the problems we know).

**Comment by ahocquard** [comment]
Closed as the project is aborted.
